app/scripts/migrations/119.test.ts

```diff
@@ -0,0 +1,96 @@
+import { createMockInternalAccount } from '../../../test/jest/mocks';
+import { migrate, version } from './119';
+
+const mockTimeStamp = 1716972230;
+jest.useFakeTimers().setSystemTime(new Date(mockTimeStamp));
+
+const oldVersion = 118;
+const newVersion = 119;
+
+describe('migration #119', () => {
```

Comment:
Should we add the test cases for when some of the check in the `if` scope fails?

```ts
if (
    hasProperty(state, 'AccountsController') &&
    isObject(state.AccountsController) &&
    hasProperty(state.AccountsController, 'internalAccounts') &&
    hasProperty(
      state.AccountsController
        .internalAccounts as AccountsControllerState['internalAccounts'],
      'accounts',
    ) &&
    Object.values(
      (state.AccountsController as AccountsControllerState).internalAccounts
        .accounts,
    ).length > 0
  )
```